nemo/collections/asr/data/audio_to_text_dali.py

```diff
@@ -550,6 +624,7 @@ def __init__(
         shuffle: bool = False,
         drop_last: bool = False,
         parser: Union[str, Callable] = 'en',
+        shard_strategy: str = "scatter",
```

Comment:
The use of " seems to be inconsistent with one line above. Should we try to be consistent in terms of use of quotation marks? I notice some inconsistencies in other parts of the file as well...

Reply:
Well strings should almost always be "" but python allows both, it's fine to try to be consistent but it's fine not follow it to the letter too. There's no pep against such mixing, not even black style comments about it.